lib/rspotify/playlist.rb

```diff
@@ -128,8 +126,8 @@ def initialize(options = {})
 
       super(options)
 
-      @path = "users/#{@owner.instance_variable_get('@id').gsub('?','')}/"
-      @path << (@href =~ /\/starred$/ ? 'starred' : "playlists/#{@id}")
+      @path = "users/#{@owner.instance_variable_get('@id').delete('?')}/"
+      @path << (@href =~ %r{\/starred$} ? 'starred' : "playlists/#{@id}")
```

Comment:
Why is `%r{}` preferred over `//`?

Reply:
I've juste followed what Rubocop told me on this but we can change the option.
It's kinda strange because the default seems to be slashes but here the default is percent
I will change it to slashes.

Reply:
It's because of the presence of an inner slash